foundry.toml

```diff
@@ -1,9 +1,10 @@
 [profile.default]
 optimizer = true
 optimizer_runs = 200
-lib-paths = "node_modules"
 contracts = "contracts"
 fuzz_max_global_rejects = 100000
 fuzz_max_local_rejects = 100000
 fuzz_runs = 500
-libs = ["node_modules", "lib"]
+# 'contracts/test', 'contracts/mock', 'forge-std/lib/ds-test/src', 'forge-std/src' are added here
+# to not appear in the code coverage result when using forge coverage
+libs = ['node_modules', 'lib', 'contracts/test', 'contracts/mock', 'forge-std/lib/ds-test/src', 'forge-std/src']
```

Comment:
I really don't know if that change can affect something else, but without that the code coverage would output the coverage for the tests files which is 0%